packages/forms-web-app/__tests__/unit/controllers/full-appeal/submit-appeal/application-form.test.js

```diff
@@ -27,6 +27,7 @@ describe('controllers/full-appeal/submit-appeal/application-form', () => {
 	const taskName = documentTypes.originalApplication.name;
 	const errors = { 'file-upload': 'Select a file upload' };
 	const errorSummary = [{ text: 'There was an error', href: '#' }];
+	const sectionTag = 'APPLICATION FORM';
```

Comment:
@guiribeirokc potentially out of scope, would it make sense to add the sectionTag to the documentTypes object in common (being used on line 27 already) and call from there for a single source of truth to reference to update should these section tags be the same between appeal types? Rather than hardcoded in each controller, only a thought though.